### DISCUSSION TITLE
Ice Ice Baby

### DISCUSSION THREAD

**Discussion by Q4ik** [unitystation/unitystation#8395] [Ideas]
### Description

Water vapor ejected into space should turn into ice. That being said, water vapor in the air should condensate on walls, glass, floors, pipes, machines (could short them out), etc...

**Comment by DTraitor** [comment]
Vacuum is not cold at all. Water vapor won't cool once it leaves the station, at least not that fast. Vapor will slowly irradiate its warm but nothing else

**Reply by PerfectTangent** [reply]
> Vacuum is not cold at all. Water vapor won't cool once it leaves the station, at least not that fast. Vapor will slowly irradiate its warm but nothing else

The state of a material is not just controlled by temperature. Pressure is also a factor, and water would readily become ice at quite a higher temperature than typical in a total vacuum. It could reasonably freeze in a few minutes.

**Reply by ilinm1** [reply]
Lower pressure also makes boiling temp lower so vacuum would have opposite effect on water

**Reply by PerfectTangent** [reply]
> Lower pressure also makes boiling temp lower so vacuum would have opposite effect on water

An 'opposite' effect would be that the water got hotter. It can't get hotter without energy being added into the system. Boiling is a symptom of the lack of pressure, and increases the surface area of water directly exposed to vacuum, greatly increasing the rate at which it would cool. The fact that the water boils in such a low pressure is _why_ it would rapidly freeze. It skips from gas to solid via deposition.
